app/controllers/api/v2/discovered_hosts_controller.rb

```diff
@@ -145,13 +152,14 @@ def auto_provision_all
               logger.warn "No rule found for host %s" % discovered_host.name
             end
           end
+
           if result
             process_success _("Discovered hosts are provisioning now")
```

Comment:
@lzap when testing the changes I noticed that the api returns an empty hash when it is successful and not this message.